### DISCUSSION TITLE
Test scenario : Fairdrive interoperability

### DISCUSSION THREAD

**Discussion by crtahlin** [fairDataSociety/fairdrive-theapp#170] [Quality assurance]
# Fairdrive interoperability - test scenario n1

Test on: Chrome; recommended also Firefox

1. Create account via https://create.dev.fairdatasociety.org
All following steps wil be done with the created account.
2. Login into Fairdrive via https://app.fairdrive.dev.fairdatasociety.org/
3. Go to Drive, create a Pod.
4. Create a folder in the pod.
5. Upload an example markdown file to the folder, it should have an .md suffix and contain markdown.
6. Go to https://app.dracula.dev.fairdatasociety.org/ and use FD Connect to connect to storage.
7. Find and open the example markdown file.
8. Edit the markdown file, save it under a different name.
9. Go to https://slidezz.link.dev.fairdatasociety.org and login
10. Create a new presentation by loading one of the existing markdown files
11. Edit the presentation
12. Save the presentation as .html
13. Check that the created presentation is in Fairdrive

**Comment by tamas6** [comment]
6. link should be https://app.dracula.dev.fairdatasociety.org/  - - we just realized it in today's short-sync call

**Reply by crtahlin** [reply]
Thank you for finding the bug in the bug testing script. Corrected.

**Comment by tamas6** [comment]
Dracula: I'm having hard time finding that markdown sample I have in fairdrive pod
![Screenshot 2022-10-10 at 17 41 36](https://user-images.githubusercontent.com/81620050/194904807-8469e5cd-44f2-48c6-abbc-7657266b02b9.png)

I can enter, and pick the pod, but cannot do more. 
tried to save a file:
![Screenshot 2022-10-10 at 17 45 43](https://user-images.githubusercontent.com/81620050/194905475-4d7534d4-12e7-4621-8745-9679c521224d.png)

the sw extension listens to a correct running mainnet node btw

**Reply by crtahlin** [reply]
Converted to issue: https://github.com/fairDataSociety/dracula.me/issues/10

**Reply by tamas6** [reply]
still stuck at 7. of test scenario
